Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	docker build -t collabtest/collabtest . && docker run -it --rm -p 8080:8080 collabtest/collabtest
```

Comment:
Good point, but you still might be running the server in a different terminal that we should kill. Shouldn't hurt, and avoids annoyances.